relay-server/src/actors/project_cache.rs

```diff
@@ -790,6 +818,11 @@ impl ProjectCacheBroker {
         self.buffer.send(spooler::Health(sender))
     }
 
+    fn handle_update_global_config(&mut self, global_config: Arc<GlobalConfig>) {
+        self.source.upstream_source.send(global_config.clone());
```

Comment:
ideally I'd send the global config directly from the 'GlobalConfigService', but the 'UpstreamProjectSource' service is created within the 'ProjectSource' service, and not in 'ServiceState::Start' so I don't have access to the Addr when creating the 'GlobalConfigService'. meaning I cannot send it directly without refactoring a bunch.